app/serializers/subject_serializer.rb

```diff
@@ -3,9 +3,9 @@ class SubjectSerializer
   include FilterHasMany
 
   attributes :id, :metadata, :locations, :zooniverse_id,
-    :created_at, :updated_at, :href
+    :created_at, :updated_at, :href, :finished_workflow
```

Comment:
any reason this came out of the optional block - it should only be serializing if the context is set (i.e. via the queued selection end point). Change it back to the way you had it for `seen_all_live_data` but keep the attribute naming you have here.